frame/support/test/tests/pallet_instance.rs

```diff
@@ -567,19 +557,9 @@ fn pallet_hooks_expand() {
 #[test]
 fn pallet_on_genesis() {
```

Comment:
What does this test now that you removed all the asserts?

Reply:
First, we always now have a version returned and if there is nothing in the storage we return the default value (0).

Secondly the almost same test exists in `pallet.rs`

Reply:
Am I going crazy? Or do I see the diff wrong?

All I see is this:

```
pallet::Pallet::<Runtime>::on_genesis();
pallet::Pallet::<Runtime, pallet::Instance1>::on_genesis();
```

This only shows me these two calls dont panic?

Reply:
Yeah, maybe I should remove this xD